micro-rdk-ffi/micrordklib-idf-component/CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
 idf_component_register(INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}")
 idf_build_set_property(INCLUDE_DIRECTORIES "${CMAKE_CURRENT_BINARY_DIR}/assets" APPEND)
 
-set(LIBMICRORDK_URL https://github.com/viamrobotics/micro-rdk/releases/download/${COMPONENT_VERSION}/micro-rdk-lib.zip)
+set(LIBMICRORDK_VERSION v0.4.1-rc4)
```

Comment:
was this line here before?

Reply:
Yes: https://github.com/viamrobotics/micro-rdk/blob/v0.4.0/micro-rdk-ffi/micrordklib-idf-component/CMakeLists.txt#L4.